docs/hub/agents.md

```diff
@@ -16,7 +16,7 @@ With the HF MCP Server, you can enhance your AI assistant's capabilities by conn
 
 #### Getting Started
 
-Visit [huggingface.co/settings/mcp](https://huggingface.co/settings/mcp) to configure your MCP client and get started.
+Visit [huggingface.co/settings/mcp](https://huggingface.co/settings/mcp) to configure your MCP client and get started. Read the dedicated one‑page guide: [HF MCP Server](./hf-mcp-server).
```

Comment:
```suggestion
Visit [huggingface.co/settings/mcp](https://huggingface.co/settings/mcp) to configure your MCP client and get started with our [short guide](./hf-mcp-server).
```

Reply:
I have a slight preference for the former over this.